### PR TITLE
Case 20879: Migrate  webEngine referencing  QML into +webengine folder

### PR DESCRIPTION
**Pull request by danteruiz**
Migrate WebEngine referencing QML into a +webengine folder, replace top level item with stubs 

Ticket - https://highfidelity.manuscript.com/f/cases/20879/Migrate-WebEngine-referencing-QML-into-a-webengine-folder-replace-top-level-item-with-stubs

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/16905/HighFidelity-Beta-PR14808-1c5a066.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/16905/HighFidelity-Beta-PR14808-1c5a066.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/16905/HighFidelity-Beta-Interface-PR14808-1c5a066.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/16905/HighFidelity-Beta-PR14808-1c5a066.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/16905/HighFidelity-Beta-Interface-PR14808-1c5a066.exe

File: interface/resources/qml/controlsUit/+webengine/WebView.qml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+//
```
Comment (jherico) [inline interface/resources/qml/controlsUit/+webengine/WebView.qml]: why did this file get moved?  It doesn't reference WebEngine.  Although it does refer to the newViewRequested signal, there's no reason that we can't make the ProxyWebView have all the same signals as the actual WebEngineView.

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/16917/HighFidelity-Beta-PR14808-801cc03.apk

File: interface/resources/qml/TabletBrowser.qml [hunk 2]
```diff
@@ -30,96 +24,8 @@ Item {
...
-        switch(event.key) {
-        case Qt.Key_L:
-            if (event.modifiers == Qt.ControlModifier) {
-                event.accepted = true
-                addressBar.selectAll()
-                addressBar.forceActiveFocus()
-            }
-            break;
-        }
+        anchors.centerIn: parent
```
Comment (jherico) [inline interface/resources/qml/TabletBrowser.qml]: why are we not using the same layout as the original?

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/16919/HighFidelity-Beta-PR14808-93a91cd.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/16919/HighFidelity-Beta-PR14808-93a91cd.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/16919/HighFidelity-Beta-Interface-PR14808-93a91cd.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/16919/HighFidelity-Beta-PR14808-93a91cd.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/16919/HighFidelity-Beta-Interface-PR14808-93a91cd.exe

**Comment by JeffClinton** [comment]
Build this please

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/17041/HighFidelity-Beta-PR14808-93a91cd.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/17041/HighFidelity-Beta-PR14808-93a91cd.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/17041/HighFidelity-Beta-Interface-PR14808-93a91cd.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/17041/HighFidelity-Beta-PR14808-93a91cd.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/17041/HighFidelity-Beta-Interface-PR14808-93a91cd.exe

**Pull request merged**